### DISCUSSION TITLE
Clarification v6 to v7 migration guide: input value and reference will no longer get removed after unmount.

### DISCUSSION THREAD

**Discussion by wondering639** [react-hook-form/react-hook-form#5292] [Q&A] [answered]
In the [v6 to v7 upgrade guide](https://react-hook-form.com/migrate-v6-to-v7/) on register it says:

> Important: input value and reference will no longer get removed after unmount.
What does that practically mean? When should I care about this?

**Comment by bluebill1049** [comment, marked as answer]
When an input gets unmounted, its value will be maintained in the formState and included in the submission
